playwright/tests/office/txo/tooFlows.spec.js

```diff
@@ -427,4 +427,44 @@ test.describe('TOO user', () => {
       await tooFlowPage.waitForPage.moveDetails();
     });
   });
+
+  test('approves a delivery address change request for an HHG shipment', async ({ officePage, page }) => {
+    const shipmentAddressUpdate = await officePage.testHarness.bulidHHGMoveWithAddressChangeRequest();
+    await officePage.signInAsNewTOOUser();
+    tooFlowPage = new TooFlowPage(officePage, shipmentAddressUpdate.Shipment.MoveTaskOrder);
+    await officePage.tooNavigateToMove(shipmentAddressUpdate.Shipment.MoveTaskOrder.locator);
+
+    await expect(page.getByText('Review required')).toBeEnabled();
+
+    // Edit the shipment
+    await page.locator('[data-testid="ShipmentContainer"] .usa-button').first().click();
+
+    await expect(page.getByText('Request needs review. See delivery location to proceed.')).toBeEnabled();
```

Comment:
Should I use toBeVisible() instead?

Reply:
Maybe https://playwright.dev/docs/api/class-locator#locator-and will let us do something like

```suggestion
    await expect(page.getByRole('alert').and(page.getByText('Request needs review. See delivery location to proceed.'))).toBeEnabled();
```

It's just a little extra assurance that the thing you're looking for is the type of element you want, rather than just confirming the text content is there